_posts/2022-05-04-RTK-Intership.md

```diff
@@ -0,0 +1,249 @@
+
+In this range there are still errors, these errors are caused by satellite clock and ephemeris, but also by ionospheric and tropospheric delays. 
+We can solve this by sending correction data from the base station to the rover over a mobile network or another type of wireless communication such as LoRa.
+### Network RTK
+Network RTK is based on the use of several fixed reference stations spread out over the network. 
+Depending on the application, the rover sends its location to the reference stations, so the rover is not dependent on one base station but can connect to several reference stations. 
+This allows for a greater range than a single fixed base station, which is about 10 km.
+### How to receive correction data?
+There are a few ways to receive this correction data. You can build your own base station or use a service. 
+Not all services are free, for example Skylark, this service has coverage over 3 continents but is paying. 
+Fortunately, the Flemish government has set up a service called FLEPOS. 
+
+## What is FLEPOS?
+Flemish Positioning Service (FLEPOS) is a service of Digital Flanders, where correction streams from navigation satellites are distributed via a mobile network, using the NTRIP protocol. 
+These correction streams make it possible to determine an accurate position anywhere in Flanders. 
+Using FLEPOS services is completely free, the connection costs over mobile networks, such as 4G, are not free.
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/FLEPOS.png' | prepend: site.baseurl }}" alt="FLEPOS" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+### Usage of FLEPOS
+FLEPOS is used within various markets that require accurate positioning:
+
+- Surveying, photogrammetry
+- Precision agriculture
+- GIS inventory
+- Hydrography
+- Machine tool control
+- Dredging
+- ...
+
+A specific RTK service has been set up for each of these markets. 
+FLEPOS also offers data for scientific research. 
+Before FLEPOS can be used, the registration form must be completed and approved on the website of Digitaal Vlaanderen.
+
+### FLEPOS mountpoint
+To receive the correction data, it is necessary to connect to a mountpoint of FLEPOS. 
+The figure below shows all available mountpoints. 
+The most commonly used data streams are FLEPOSVRS31GR and FLEPOSVRS32GREC. 
+The user is free to take one of the other available data streams for the GNSS receiver. 
+Data stream FLEPOSMARIVRS31GR is recommended only for measurement in the North Sea because this data stream uses the 2 overseas stations ALDB and SHOE and the station BLIG located on the Bligh sandbank.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/Mounpoints.png' | prepend: site.baseurl }}" alt="FLEPOS" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+## RTK module and antenna
```

Comment:
Explain that you'll now go into the implementation of a system and why you choose this system.